content/docs/user-guide/gems/repositories/overview.md

```diff
@@ -0,0 +1,194 @@
+---
+linkTitle: Overview
+title: Gem Repository Overview
+description: Gems repositories provide a way to share Gems in Open 3D Engine (O3DE).
+weight: 100
+toc: true
+---
+
+
+Gem repositories provide a way to discover, acquire, and share Gems that are not included with **Open 3D Engine (O3DE)**.  Gem repositories are generally hosted online, but you can also add them to your project from a local hard drive which is helpful for testing.
+
+
+## Add a Gem repository in Project Manager
+
+You can add Gem repositories from the **Gem Repositories** page in Project Manager.
+1. Press the **Add Repository** button. In the dialogue box that opens you can supply the URL or local path for a Gem repository. 
+1. If you have a URL for an online Gem repository, copy and paste it into the `Repository Path` field and press the **Add** button.
+1. Alternately, if you want to add a Gem repository from your local hard drive, press the folder button to open the `Browse` dialog box, select the folder with the Gem repository, press the **Select Folder** button, and finally press the **Add** button.
+
+A new entry will be added to the list of Gem repositories, and the Gems provided by the Gem repository will be available to download and add to your projects.
+
+## Add a Gem repository with the o3de command line tool
+
+You can use the `o3de` CLI tool to add Gem repositories with the following command:
+```
+scripts\o3de.bat register -ru <Gem repository URL>
+```
+
+## Anatomy of a Gem repository  
+
+A Gem repository contains a `repo.json` file which contains the metadata for the repository, including a list of URIs for the Gems it provides.
+
+### Folder structure
+
+The recommended folder structure is to put the `repo.json` file at the root of your Gem repository and the `gem.json` files for each Gem it provides in subfolders.  
+```
+/
+   repo.json
+   ExampleGem1/
+       gem.json
+   ExampleGem2/
+       gem.json
+```
+
+If you are confident you will only ever have one Gem in the repository, it's acceptable to put the `gem.json` in the root folder.
+
+{{< note >}}
+One of the most convenient ways to host a Gem Repository online is to place the `repo.json` file inside the GitHub repository where your Gem's source code is.
+{{< /note >}}
+
+### Format of a repo.json file
+The `repo.json` file contains the metadata for the Gem Repository and a list of Gems.
+
+```
+{
+    "repo_name":"RepositoryName",
+    "origin":"RemoteOriginName",
+    "repo_uri": "http://localhost:8080/",
+    "summary": "Example Repository",
+    "additional_info": "Example repository hosted on local http server.",
+    "last_updated": "10/26/2021 04:56pm",
+    "gems": [
+        "http://localhost:8080/ExampleGem/"
+    ]
```

Comment:
Since `localhost` is only used for testing, I don't think it should appear in the format examples. We should provide an example of the most common format - perhaps GitHub? So maybe use something like `"https://github.com/MyRepo"` and `"https://github.com/MyRepo/ExampleGem/"`.